### ISSUE TITLE
Accessibility: names of dashboards in sidebar not shown to screen readers in Google Chrome

### ISSUE THREAD

**Issue by Timtam** [home-assistant/home-assistant-polymer#27913]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [x] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Since  about updating to 2025.10, NVDA within Google Chrome doesn't pick up names of dashboards within the sidebar anymore. In Firefox it seems to still work though. It also kinda returned to normal for a while too, but since updating to 2025.11 it doesn't work anymore again. I've got a feeling it might be related to the size of the window, probably when its too small.

### Describe the behavior you expected

Screen readers should always be able to see the name of the dashboards within the sidebar, no matter the resolution of the browser window.

### Steps to reproduce the issue

1. Run a screen reader, like the free and open-source NVDA.
2. Open Home Assistant and log in.
3. Scroll through the dashboards within the side bar.
4. Try resizing the browser window and retry. At some point, you will see that dashboard elements get announced as "clickable" instead of showing their names.

### What version of Home Assistant Core has the issue?

2025.11.1

### What was the last working version of Home Assistant Core?

2025.9.* I think

### In which browser are you experiencing the issue?

Google Chrome Version 142.0.7444.60

### Which operating system are you using to run this browser?

Windows 11 24H2

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### JavaScript errors shown in your browser console/inspector

```txt

```
